packages/snap/src/index.ts

```diff
@@ -438,6 +438,31 @@ export const onRpcRequest = async ({
       return await vcManager.getAllDecryptedVCs();
     }
 
+    case RPCMethods.ExportIdentity: {
+      if (!GET_CREDENTIALS_SUPPORTED_HOSTNAMES.includes(getHostname(origin))) {
+        throw new Error('This origin does not have access to credentials');
+      }
```

Comment:
maybe new whitelist supposed to be used only for export purposes? We need it only on dashboard